lib/ui/login/login_screen.dart

```diff
@@ -48,36 +69,124 @@ class _LoginScreenState extends State<LoginScreen>
       }
     });
 
+    _logoPositionAnimationController.addStatusListener((status) {
+      if (status == AnimationStatus.completed) {
+        _overlayOpacityAnimationController.forward();
+      }
+    });
+
     _logoOpacityAnimationController.forward();
   }
 
-  Stack _buildLoginScreen() => Stack(
+  TextStyle? _textFieldTextStyle(BuildContext context) =>
+      Theme.of(context).textTheme.bodySmall;
+
+  EdgeInsets _textFieldEdgeInsets() => const EdgeInsets.only(
+        left: AppDimensions.spacing24,
+        right: AppDimensions.spacing24,
+        bottom: AppDimensions.spacing20,
+      );
+
+  EdgeInsets _buttonEdgeInsets() => const EdgeInsets.symmetric(
+        horizontal: AppDimensions.spacing24,
+      );
```

Comment:
Optional: Since they are only being used once, it's unnecessary to create a separate variable for them